creator-node/src/services/sync/processSync.js

```diff
@@ -6,6 +6,48 @@ const { saveFileForMultihashToFS } = require('../../fileManager')
 const { getOwnEndpoint, getCreatorNodeEndpoints } = require('../../middlewares')
 const SyncHistoryAggregator = require('../../snapbackSM/syncHistoryAggregator')
 const DBManager = require('../../dbManager')
+const CIDFailureCountManager = require('./CIDFailureCountManager')
+
+/**
+ * Wrapper around `saveFileForMultihash()` with CID failure count tracking
+ * Throws error if CID failure count is below threshold, else returns boolean indicating saved or skipped
+ * @returns {Boolean} skipped
+ */
+async function saveFileForMultihashToFSWrapper ({
```

Comment:
I'd rename this to skippableSaveFileForMultihash or something more indicative

Reply:
much better, done